loom/engine/bindings/loom/lmApplication.cpp

```diff
@@ -85,15 +85,17 @@ void loom_appSetup(void)
 
 void loom_appPause(void)
 {
-    atomic_store32(&gLoomTicking, 0);
+    int ticking = atomic_decrement(&gLoomTicking);
+    if (ticking != 0) return;
     GFX::Graphics::pause();
     lmLogInfo(applicationLogGroup, "Paused");
 }
     
 void loom_appResume(void)
 {
+    int ticking = atomic_increment(&gLoomTicking);
```

Comment:
I think it is as the only place it's written is here (and the decrement, which is called at a different time on the same thread), so it doesn't need a write lock, but it does need to be atomic so that reads in `lmApplicationTasks` don't read a corrupted value.